app/controllers/dashboard_controller.rb

```diff
@@ -1,3 +1,9 @@
 class DashboardController < ApplicationController
+  before_action :show_sidebar
+
   def index; end
+
+  def show_sidebar
```

Comment:
chỉ dùng trong class thì khai báo thêm private nha em
```suggestion
  private
  def show_sidebar
```